packages/venia-ui/lib/RootComponents/Category/NoProductsFound/noProductsFound.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { string, shape, objectOf, number } from 'prop-types';
+import { mergeClasses } from '../../../classify';
+import { Link, resourceUrl } from '@magento/venia-drivers';
+import defaultClasses from './noProductsFound.css';
+
+const NoProductsFound = props => {
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const categories = props.categories;
+
+    // TODO: get categoryUrlSuffix from graphql storeOptions when it is ready
+    const categoryUrlSuffix = '.html';
```

Comment:
Are you sure `html` is not needed anymore since `packages/venia-ui/lib/components/CategoryList/categoryTile.js` is also doing it the same way as i did ? 
If `.html` is default now we don't need the suffix anymore i suppose ?

Reply:
You are right. We don't need a suffix any more but it's your call at this point. If it makes sense to have it for now, since other components do, just move it out of the component since it is a constant.